test/OpenSpaceToolkit/Astrodynamics/Flight/Profile.test.cpp

```diff
@@ -34,99 +37,101 @@
-            {
-                return Transform::Identity(anInstant) ;
-            }
-        } ;
 
-        const Shared<const Frame> frameSPtr = Frame::GCRF() ;
+            using ostk::core::types::Real ;
 
-        EXPECT_NO_THROW(Profile profile(dynamicTransformProvider, frameSPtr)) ;
+            using ostk::physics::units::Length ;
+            using ostk::physics::units::Angle ;
+            using ostk::physics::units::Derived ;
+            using ostk::physics::time::Scale ;
+            using ostk::physics::time::Instant ;
+            using ostk::physics::time::DateTime ;
+            using ostk::physics::coord::Frame ;
+            using ostk::physics::Environment ;
+            using ostk::physics::env::obj::celest::Earth ;
 
-    }
+            using ostk::astro::trajectory::orbit::models::Kepler ;
+            using ostk::astro::trajectory::orbit::models::kepler::COE ;
+            using ostk::astro::flight::Profile ;
 
-}
+            const Environment environment = Environment::Default() ;
 
-TEST (OpenSpaceToolkit_Astrodynamics_Flight_Profile, StreamOperator)
-{
+            const Length semiMajorAxis = Length::Kilometers(7000.0) ;
+            const Real eccentricity = 0.0 ;
+            const Angle inclination = Angle::Degrees(0.0) ;
+            const Angle raan = Angle::Degrees(0.0) ;
+            const Angle aop = Angle::Degrees(0.0) ;
+            const Angle trueAnomaly = Angle::Degrees(0.0) ;
 
-    using ostk::core::types::Shared ;
+            const COE coe = { semiMajorAxis, eccentricity, inclination, raan, aop, trueAnomaly } ;
 
-    using ostk::physics::time::Instant ;
-    using ostk::physics::coord::Transform ;
-    using ostk::physics::coord::Frame ;
-    using ostk::physics::coord::frame::provider::Dynamic ;
+            const Instant epoch = Instant::DateTime(DateTime(2018, 1, 1, 0, 0, 0), Scale::UTC) ;
+            const Derived gravitationalParameter = Earth::GravitationalParameter ;
+            const Length equatorialRadius = Earth::EquatorialRadius ;
+            const Real J2 = Earth::J2 ;
+            const Real J4 = Earth::J4 ;
 
-    using ostk::astro::flight::Profile ;
+            const Kepler keplerianModel = { coe, epoch, gravitationalParameter, equatorialRadius, J2, J4, Kepler::PerturbationType::None } ;
 
-    {
+            this->orbit_ = { keplerianModel, environment.accessCelestialObjectWithName("Earth") } ;
 
-        const Dynamic dynamicTransformProvider =
-        {
-            [] (const Instant& anInstant) -> Transform
-            {
-                return Transform::Identity(anInstant) ;
-            }
-        } ;
+            this->profile_ = Profile::NadirPointing(this->orbit_, Orbit::FrameType::VVLH) ;
 
-        const Shared<const Frame> frameSPtr = Frame::GCRF() ;
+        }
 
-        const Profile profile = { dynamicTransformProvider, frameSPtr } ;
+        Orbit orbit_ = Orbit::Undefined() ;
+        Profile profile_ = Profile::Undefined() ;
```

Comment:
Needed?

Reply:
Yes because the fixture data only gets initialized in the `SetUp` method, so before that objects needs to be constructed (and `Profile profile_ ;` only is not valid))

Reply:
Makes sense!